src/web/utils/errorHelpers.ts

```diff
@@ -10,15 +10,15 @@ export const analyticsIdentifier = (errorId?: string, errorHash?: string) => {
 export const extractMessageFromAxiosError = (err: Error) => {
   // message does not exist on the data field for Axios Error.Response, but we add it
   // in our API for custom messages
-  let message = null;
   if (axios.isAxiosError(err)) {
     if (err.response?.data?.message) {
```

Comment:
Just updating this so the return type is string|undefined (instead of string|null) so I can use it in component props.